lale/lib/lale/map.py

```diff
@@ -29,6 +29,8 @@ def transform(self, X):
         def get_map_function_output(column, new_column_name):
             functions_module = importlib.import_module("lale.lib.lale.functions")
             function_name = column._expr.func.id
+            if _is_ast_attribute(column):
```

Comment:
is the line to support `columns={"new": it.old}` (without an explicit rename?)  if so, that makes sense, but please add a test for it

Reply:
Na.. Actually left it by mistake. I will remove it as we want only explicit rename.